### DISCUSSION TITLE
scan error

### DISCUSSION THREAD

**Discussion by mayamaniac** [photoview/photoview#292] [Q&A and Troubleshooting]
`Could not delete old albums from database: Error 1451: Cannot delete or update a parent row: a foreign key constraint fails (`photoview`.`user_albums`, CONSTRAINT `fk_user_albums_album` FOREIGN KEY (`album_id`) REFERENCES `albums` (`id`)) `

Some folders were deleted and the scanner is giving me this error and now it won't update the scan. How do I fix this?

**Comment by viktorstrate** [comment]
See related issue #198. For now you could try to delete the path from the user and add it again, that should clean up the database.
